doc/api/net.md

```diff
@@ -138,7 +138,8 @@ server.listen(() => {
 });
 ```
 
-Don't call `server.address()` until the `'listening'` event has been emitted.
+If you call `server.address()` before the `'listening'` event has been emitted,
+or after `server.close()` has been called, `null` will be returned.
```

Comment:
```suggestion
If `server.address() is called before the `'listening'` event has been emitted,
or after `server.close()` has been called, `null` will be returned.
```

Reply:
+1 on removing _you_, but how about we replace it with active voice rather than passive voice? And keep it nice and short?

```suggestion
`server.address()` returns `null` before the `'listening'` event or after
calling `server.close()`.
```

Reply:
@Trott I find your version a bit harder for my brain to parse, but maybe it's just me. But if I edit it slightly it's easier for me:

```suggestion
`server.address()` returns `null` before the `'listening'` event has been emitted
or after calling `server.close()`.
```

Reply:
@watson To me that sounds good, do you want to apply it?